lib/Xud.ts

```diff
@@ -50,15 +51,16 @@ class Xud {
       this.db = new DB(this.config.db);
       await this.db.init();
 
-      this.lndClient = new LndClient(this.config.lnd);
-      await this.lndClient.connect();
```

Comment:
Restore this `connect` statement (for both lnds), we want it to be called from here and not from the constructor so we can properly wait for it.

Reply:
done